### ISSUE TITLE
Failed formatting for this JSON object

### ISSUE THREAD

**Issue by Phrogz** [Phrogz/NeatJSON#6]
~~~js
var obj = '{ "abcdefghij":[{ "abcdefghijklmnop":{} }] }';
var json = neatJSON(obj, {wrap:1, short:true, aroundColonN:1} );
// TypeError: Cannot read property '0' of undefined
~~~

Removing characters from either property name fixes the problem, as does removing the `aroundColonN` option. (?!)

**Comment by Phrogz** [comment]
Simpler repro:

~~~js
var obj = {"foo":[{}]};
var json = neatJSON(obj, {wrap:1, short:true, aroundColonN:1} );
// TypeError: Cannot read property '0' of undefined
~~~

**Comment by Phrogz** [comment]
Fixed in v0.6.1
